contribute.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "DXR",
+    "description": "An intelligent source code browser.",
```

Comment:
"Mozilla's code search and navigation tool, aimed at making sense of large projects like Firefox"